mods/ts/rules/nod-support.yaml

```diff
@@ -270,6 +271,19 @@ NASAM:
 NASTLH:
 	Inherits: ^Building
 	Inherits@IDISABLED: ^DisabledOverlay
+	HitShape:
+		UseOccupiedCellsOffsets: false
```

Comment:
Is there a particular reason why every time I see a composite setup in the PRs, this ends up disabled? I couldn't notice anything wrong if I used additional hitboxes without targetable offsets.